### DISCUSSION TITLE
Adding tiered pricing

### DISCUSSION THREAD

**Discussion by LCooper-Kencove** [saleor/saleor#8376] [Q&A]
We are hoping to add support for tiered pricing. We offer price breaks on items if people order a certain amount. For example, an item normally has a price of $5.00, but if you buy 10 of them you'll only pay $4.50 per item for a total of $45. 

We're not sure what the best route would be for implementation and are looking for suggestions. Our options seem to be modifying the existing code, creating an app, or creating a plugin. At the moment we are leaning towards a plugin. 

We'd like for these values to be editable in the dashboard. Is that possible with a plugin?

**Comment by PlusA2M** [comment]
Hi there! We've similar scenario even with more complicated requirements, you may checkout [here the answer from the team](https://github.com/saleor/saleor/discussions/7718). I think modifying the code of the `saleor-dashboard` is needed if you want to change how you setup the tiered pricing, our team was considering to just use the `metadata` to set it up so we could develop a plugin without touching the code of it, in a hacky way. 🤣

**Reply by mgaraiondo** [reply]
Hi,
We also have a similar scenario. Have you managed to develop any solution? What has been the strategy?
We are thinking of using the metadata to store the variant price variables.
But how have you updated the price of the order line?
Thanks in advance
